bin/node/runtime/src/lib.rs

```diff
@@ -469,37 +450,39 @@ impl pallet_staking::Trait for Runtime {
 	type SlashCancelOrigin = pallet_collective::EnsureProportionAtLeast<_3, _4, AccountId, CouncilCollective>;
 	type SessionInterface = Self;
 	type RingCurrency = Balances;
-	type RingRewardRemainder = ();
-	type RingSlash = ();
+	type RingRewardRemainder = Treasury;
+	// send the slashed funds to the treasury.
+	type RingSlash = Treasury;
+	// rewards are minted from the void
 	type RingReward = ();
 	type KtonCurrency = Kton;
-	type KtonSlash = ();
+	// send the slashed funds to the treasury.
+	type KtonSlash = Treasury;
+	// rewards are minted from the void
 	type KtonReward = ();
 	type Cap = Cap;
 	type TotalPower = TotalPower;
 	type GenesisTime = GenesisTime;
 }
 
 parameter_types! {
-	pub const EthMainet: u64 = 0;
-	pub const EthRopsten: u64 = 1;
+	pub const ProposalBond: Permill = Permill::from_percent(5);
+	pub const ProposalBondMinimum: Balance = 1 * COIN;
+	pub const SpendPeriod: BlockNumber = 1 * DAYS;
+	pub const Burn: Permill = Permill::from_percent(50);
 }
 
-impl pallet_eth_relay::Trait for Runtime {
-	type Event = Event;
-	type EthNetwork = EthRopsten;
-}
-
-impl pallet_eth_backing::Trait for Runtime {
+impl pallet_treasury::Trait for Runtime {
+	type RingCurrency = Balances;
+	type KtonCurrency = Kton;
+	type ApproveOrigin = pallet_collective::EnsureMembers<_4, AccountId, CouncilCollective>;
+	type RejectOrigin = pallet_collective::EnsureMembers<_2, AccountId, CouncilCollective>;
```

Comment:
Polkadot Codebase are using EnsureProportion, might be more suitable for production?

Another question, how the council get elected?
```
	type ApproveOrigin = collective::EnsureProportionAtLeast<_3, _5, AccountId, CouncilCollective>;
	type RejectOrigin = collective::EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>;
```

Reply:
https://github.com/paritytech/substrate/blob/master/frame/collective/src/lib.rs#L20

---

elect through https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/lib.rs